resources/deployments/mongo/device_deployments.go

```diff
@@ -198,6 +199,30 @@ func (d *DeviceDeploymentsStorage) UpdateDeviceDeploymentStatus(deviceID string,
 	return *old.Status, nil
 }
 
+func (d *DeviceDeploymentsStorage) UpdateDeviceDeploymentLogAvailability(deviceID string, deploymentID string, log bool) error {
+	// Verify ID formatting
+	if govalidator.IsNull(deviceID) ||
+		govalidator.IsNull(deploymentID) {
+		return ErrStorageInvalidID
+	}
+
+	session := d.session.Copy()
+	defer session.Close()
+
+	selector := bson.M{
+		StorageKeyDeviceDeploymentDeviceId:     deviceID,
+		StorageKeyDeviceDeploymentDeploymentID: deploymentID,
+	}
+
+	update := bson.M{"$set": bson.M{StorageKeyDeviceDeploymentIsLogAvailable: log}}
```

Comment:
Update formatting so that it looks like this:
```
bson.M{
    "$set": bson.M{
        StorageKeyDeviceDeploymentIsLogAvailable: log
     }
}
```